Migration script/openIMIS migration v1.4.1 - v1.4.2.sql

```diff
@@ -1514,3 +1514,11 @@ AND LocationType = N'D'
 GO
 
 CREATE UNIQUE CLUSTERED INDEX CI_tblDistricts ON tblDistricts(DistrictId)   
+
+
+-- OP-153: Additional items in the Reports section
```

Comment:
Please move this in the new migration file. See comment on OTC-111 PR.

Reply:
Moved to  `Migration script/openIMIS migration v1.4.2 - tbd.sql`